Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -1490,6 +1514,44 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
             Y.Mock.verify(this.userMock);
             Y.Mock.verify(this.capiMock);
         },
+
+
+        "Should launch 'logOut' event when logging out": function () {
+            var sentLogOut = false,
+                logOutResponse = {},
+                logOutError = {};
+
+            Y.Mock.expect(this.capiMock, {
```

Comment:
the configuration of the mocks is exactly the same as in `"Should reset the user and provide the response when logging out`, so please put that in dedicated methods instead of duplicating the code.